.github/workflows/release.yaml

```diff
@@ -33,14 +33,20 @@ jobs:
           charts_dir: /home/runner/work/graphistry-helm/graphistry-helm/charts
           config: .github/config/chart-releaser.yaml
 
+      - name: Get Chart version
+        id: get_chart_version
+        uses: mikefarah/yq@master
+        with:
+          cmd: echo "TAG_NAME=$(cat graphistry-helm/charts/graphistry-helm/Chart.yaml | yq .version)" >> $GITHUB_ENV
+
```

Comment:
out of curiosity, why do we need to add the tag name to this env var `GITHUB_ENV`? we are not using `GITHUB_ENV` directly in other parts of the code right?

Reply:
because this is getting the chart version number so we can keep the chart bundle in line, it wouldnt accept github.refs so i use yq to parse the chart.yaml for the version number, this is the secure way to share env vars between diff actions